react/components/CustomTags.tsx

```diff
@@ -1,111 +1,148 @@
 import { coy } from 'react-syntax-highlighter/dist/styles/prism'
 import AnchorLink from 'react-anchor-link-smooth-scroll'
 
 import { useRuntime } from 'vtex.render-runtime'
 
-const customParagraph: FunctionComponent = ({ children }) => (
-  <p className="t-body lh-title">{children}</p>
-)
+export const CustomRenderers = {
+  break: () => <br />,
+  code: (props: any) => {
+    const codeBlock = props.value.replace(/“/gm, '"').replace(/”/gm, '"')
 
-const customH1: FunctionComponent = ({ children }) => (
-  <h1
-    id={`${children}`.toLowerCase().replace(' ', '-')}
-    className="t-heading-2 bb b--muted-4">
-    {children}
-  </h1>
-)
-
-const customH2: FunctionComponent = ({ children }) => (
-  <h2
-    id={`${children}`.toLowerCase().replace(' ', '-')}
-    className="t-heading-3">
-    {children}
-  </h2>
-)
-
-const customH3: FunctionComponent = ({ children }) => (
-  <h3
-    id={`${children}`.toLowerCase().replace(' ', '-')}
-    className="t-heading-4">
-    {children}
-  </h3>
-)
-
-const customH4: FunctionComponent = ({ children }) => (
-  <h4
-    id={`${children}`.toLowerCase().replace(' ', '-')}
-    className="t-heading-5">
-    {children}
-  </h4>
-)
-
-const customH5: FunctionComponent = ({ children }) => (
-  <h5
-    id={`${children}`.toLowerCase().replace(' ', '-')}
-    className="t-heading-6">
-    {children}
-  </h5>
-)
-
-const customTableHeader: FunctionComponent = ({ children }) => (
-  <th className="tc ph4">{children}</th>
-)
-
-const customTableData: FunctionComponent = ({ children }) => (
-  <td className="tc pv4 ph4">{children}</td>
-)
-
-const customPre: FunctionComponent = ({ children }) => (
-  <pre>
-    <SyntaxHighlighter language="javascript" style={coy}>
-      {children}
-    </SyntaxHighlighter>
-  </pre>
-)
-
-const customListItem: FunctionComponent = ({ children }) => (
-  <li>
-    <p className="t-body lh-title">{children}</p>
-  </li>
-)
-
-const customAnchor: FunctionComponent<AnchorHTMLAttributes<any>> = ({
-  href,
-  children,
-}) => {
-  /* eslint-disable react-hooks/rules-of-hooks */
-  const { route } = useRuntime()
-
-  const isIdLink = !!href && href[0] === '#'
-  const isRelativeLink = !!href && href[0] === '/'
-
-  if (isIdLink && !!href) {
     return (
-      <AnchorLink offset={() => 80} href={href.toLowerCase()}>
-        {children}
-      </AnchorLink>
+      <div className="overflow-scroll-gradient">
+        <pre className="w-100 overflow-y-auto overflow-scroll-gradient__scroller">
+          <SyntaxHighlighter language="javascript" style={coy}>
+            {codeBlock}
+          </SyntaxHighlighter>
+        </pre>
+      </div>
     )
-  }
-
-  if (isRelativeLink) {
-    return <a href={`/docs/${route.params.app}${href}`}>{children}</a>
-  }
+  },
+  emphasis: (props: any) => <em className="i">{props.children}</em>,
+  heading: (props: any) => {
+    const hashId = `${props.children}`.toLowerCase().replace(' ', '-')
```

Comment:
Does it handle titles like: "Avião do Pará", that is, titles with accents?

If it doesnt, check this:
https://github.com/vtex-apps/store-components/blob/master/react/components/SKUSelector/utils/index.js#L130-L134

Reply:
HAHAHAHA did not think about that, awesome example

Reply:
<img width="1114" alt="Screen Shot 2019-07-17 at 3 35 40 PM" src="https://user-images.githubusercontent.com/27777263/61402020-92a88300-a8a8-11e9-9b3d-021bab7fe94a.png">

Reply:
Now it does! Thanks ;)